src/test/java/Homework18.java

```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+        public class Homework18 extends BaseTest {
+        @Test
+        public void playSong() throws InterruptedException {
+
+            navigateToPage();
+            provideEmail("john.wolschleger@testpro.io");
+            providePassword("te$t$tudentb49");
+            clickSubmit();
+            Thread.sleep(2000);
+            clickPlayNextSongBtn();
+            clickPlaySongBtn();
+
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**